src/components/categories/Categories.js

```diff
@@ -1,6 +1,34 @@
-export const Categories = () => { 
+import { useState, useEffect } from "react"
+import { useNavigate } from "react-router-dom"
+import { getCategories } from "../../managers/categories"
+import { Category } from "./Category"
+import "./Categories.css"
 
-    return( <>
-        <h1>Categories</h1>
-    </>)
+export const Categories = () => {
+
+    const [categories, setCategories] = useState([])
+    const navigate = useNavigate()
+
+    useEffect(
+        () => {
+            getCategories().then((categoryData) => {
+                setCategories(categoryData)
+            })
+        }, [])
+
+    const sortCategoryList = [...categories].sort((a, b) => a.label > b.label ? 1 : -1)
```

Comment:
It would be better to do this sorting on the server instead of the client

Reply:
Changes made.